src/samplers/tcp/stat.rs

```diff
@@ -206,7 +206,7 @@ impl TcpStatistic {
             name: "tcp_finish_connect".to_string(),
             handler: "trace_finish_connect".to_string(),
             probe_type: ProbeType::Kernel,
-            probe_location: ProbeLocation::Return,
+            probe_location: ProbeLocation::Entry,
```

Comment:
On Kernel 5.4, it does not matter if we attach this at Entry or Return.
However on Kernel 5.10, if we attach it to Return, the sock* sk can't be used to get the stats from the map as the address are moved 128 bits. Now sure about why.
But using Entry here fix the issue.